app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt

```diff
@@ -177,7 +175,7 @@ class MetadataEditingFragmentTest {
             navigateToCreateDatasetFragment()
             onView(withId(R.id.categories_editing_menu_info)).perform(click())
             waitFor(1) // increase if needed
```

Comment:
Would be nice to also test that the alert dialog is actually displayed. (I think I did test this in the learningSettingsFragment, so have a look there #235)

Reply:
Oh yes ty you're right. i'll add it here and in ImageSetTest too :=)